tests/mesa3d/desktop-intel/checkout.sh

```diff
@@ -6,7 +6,7 @@ set -xe
 DEST="$1"
 SCRIPT_DIR="$(dirname "$(realpath "${BASH_SOURCE[0]}")")"
 
-bash "${SCRIPT_DIR}/../../utils/checkout.sh" https://gitlab.freedesktop.org/mattst88/mesa a978f5818e203ecddf199038bedf55443ab3b466 "${DEST}/vendor/google/mesa_desktop_intel"
+bash "${SCRIPT_DIR}/../../utils/checkout.sh" https://gitlab.freedesktop.org/mattst88/mesa a978f5818e203ecddf199038bedf55443ab3b466 "${DEST}/vendor/google/graphics/mesa3d/desktop-intel"
```

Comment:
I think you are using another sha1 looking at the Android.bp file.

Reply:
I was. I've updated the `android` branch of https://gitlab.freedesktop.org/mattst88/mesa/ now and updated this commit to check out `b9f0775a9e4 ("Remove android/")`.